dhis-2/dhis-api/src/main/java/org/hisp/dhis/feedback/ErrorCode.java

```diff
@@ -59,6 +59,9 @@
     E2011( "User is not allowed to read data for attribute option combo: `{0}`" ),
     E2012( "User is not allowed to view org unit: `{0}`" ),
     E2013( "At least one data set must be specified" ),
+    E2014( "Unable to parse the filter `{0}`" ),
+    E2015( "Unable to parse the order param: `{0}`" ),
+    E2016( "Unable to parse `{0}`. The values available are: {1}" ),
```

Comment:
Add a bit more info about what we actually are unable to parse.

Reply:
Done